plugin/trino-druid/src/test/java/io/trino/plugin/druid/BaseDruidConnectorTest.java

```diff
@@ -128,6 +132,15 @@ public void testShowColumns()
         assertThat(query("SHOW COLUMNS FROM orders")).matches(getDescribeOrdersResult());
     }
 
+    @Test
+    public void testDriverBehaviorForStoredUpperCaseIdentifiers()
+            throws SQLException
+    {
+        DatabaseMetaData metadata = druidServer.getConnection().getMetaData();
+        // if this fails we need to revisit the DruidRemoteIdentifierSupplier implementation since the driver has probably been fixed - today it returns true even though identifiers are stored in lowercase
+        assertTrue(metadata.storesUpperCaseIdentifiers(), "Druid DatabaseMetaData#storesUpperCaseIdentifiers did not return expected value");
```

Comment:
`assertThat` instead of `assertTrue` for better error messages. And long term we want to move away from TestNG anyway so using AssertJ has that benefit as well.